README.md

```diff
@@ -3,30 +3,85 @@
-CSS-Grid, Flexbox, Media Queries, a naming system like BEM, semantic HTML tags, or other modern techniques! You may not use frameworks or libraries such as Bootstrap.
+For this project you must use Native (Vanilla/Plain) HTML5, CSS, and JavaScript
+for this project. You are encouraged to use CSS-Grid, Flexbox, Media Queries, a
+naming system like BEM, semantic HTML tags, or other modern techniques! 
 
-General instructions for all Pre-Work Projects are located in the Chingu Voyage Handbook (URL posted in the
-`#read-me-first` channel on Discord). For detailed requirements checkout the ZenHub Board that is part of this repo.
+> You may **NOT** use frameworks or libraries such as Bootstrap.
 
-![ZenHub Screenshot](./assets/ZenHub_screenshot.png)
+General instructions for all Pre-Work Projects can also be found in the [Chingu
+Library](https://voyage.docs.chingu.io/prework/howwork).
 
-To see the ZenHub tab on this GitHub repo webpage requires that you install ZenHub's browser extension. For more
-information visit the [ZenHub Browser Extension page](https://www.zenhub.com/extension) and these [installation instructions](https://t.ly/3v8w).
+### Requirements
 
-If you would like to add these tasks to your repo so you can track the progress of your Solo Project and so they will be 
-more readily available for reference you will need to manually add new issues to your ZenHub board and copy/paste the text
-in our issues to yours.
+*Structure*
+- [ ] Explore the [Conquer Template demo](https://www.free-css.com/free-css-templates/page196/conquer), then download it (FREE) to your computer
+- [ ] Make files for your app, such as an `index.html`, stylesheet, and `.js` file.
+
+      You may download the Conquer template and use its `index.html` for testing. 
+      This will allow you to compare what you've implemented against the 
+      screenshot we've provided. However, you'll need to come up with your 
+      own implementation of the CSS and JS.
+
+- [ ] Make sure you have HTML containers for each section. For instance a 
+navigation, header, footer, sections, divs
+- [ ] Footer section with your developer information
+
+*Style*
+- [ ] Cloning means to create a web page with a layout as identical as possible
+to the template. 
+
+      Fonts, colors, and images are not important. The layout of the web page
+      on the other hand should be the same. A suggestion would be to compare
+      the way the template looks face to face to the clone.
+
+- [ ] Choose some images for your template sections at [Unsplash](https://unsplash.com/) 
+or another free image/video site
+- [ ] Which fonts and colors do you want to use? Add these and media queries 
+(to look good on mobile) to your stylesheet, which you need to add to your 
+`index.html`
+- [ ] For the card sections of the page, categories, etc. decide on a layout 
+strategy and use it consistently, such as CSS-Grid or Flex
```

Comment:
'such as CSS-Grid or Flex' => 'such as CSS Grid and/or Flexbox' . This way it won't mislead a beginning reader into thinking grid/flex are used independent of each other.

Reply:
Resolved